### PR TITLE
feat: Add validation for techniquePicker 

### PR DESCRIPTION
**Pull request by deepaksftc**
Part of https://github.com/UserOfficeProject/issue-tracker/issues/1097

### CODE REVIEW

File: packages/validation/src/Questionary/techniquePicker.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        (number | undefined)[] | null | undefined
+      >
+    | Yup.NumberSchema<number | null | undefined>;
+
+  if (config.isMultipleSelect) {
+    schema = Yup.array().of(Yup.number()).nullable();
+
+    if (config.required) {
+      schema = schema.required().min(1);
+    }
```
Comment (mutambaraf) [inline packages/validation/src/Questionary/techniquePicker.ts:20]: On this do we not set the minimum number of selections.

Comment (deepaksftc) [reply packages/validation/src/Questionary/techniquePicker.ts:20]: We are not setting any minimum number of selection while creating technique picker question. As of now, it's supporting minimum of 1 selection for the question.

Comment (mutambaraf) [reply packages/validation/src/Questionary/techniquePicker.ts:20]: One last question are we not supposed to include the message here or we will handle it somewhere else.

File: packages/validation/src/Questionary/techniquePicker.ts [hunk 2]
```diff
@@ -16,13 +16,13 @@ export const techniquePickerValidationSchema = (field: any) => {
     schema = Yup.array().of(Yup.number()).nullable();
 
     if (config.required) {
-      schema = schema.required().min(1);
+      schema = schema.min(1, 'Please choose a technique');
```
Comment (mutambaraf) [inline packages/validation/src/Questionary/techniquePicker.ts:19]: Maybe 'Please select a technique' will sound better.

**Review by martin-trajanovski — APPROVED** [review]

**Review by mutambaraf — APPROVED** [review]

**Pull request merged**